Updates 3

Purple906 opened the pull request.

Purple906 commented:
@mysterycommand any thoughts on this? The error in the log is pointing to the localhost:8888 url, so I'm going to assume its erroring since thats only available on my local machine? Do I need to add that URL to the .env file to make it work?

mysterycommand commented on pages/index.js:
```diff
@@ -3,32 +3,55 @@ import Head from "next/head";
+}
+
+function getWordPress() {
+  const wordPressApi = "http://localhost:8888/wp-json/wp/v2";
```
Oh yeah, `localhost:8888` only exists inside your computer. In order for this to work on Netlify/in the build you’ll have to have a WordPress instance up and running somewhere on the web. Then you'll change this line to be like (just an example):

```ts
const wordPressApi = "https://cog-brews.wordpress.com/wp-json/wp/v2";
```

The name `cog-brews.wordpress.com` is just a stand in, it could be anything really, just as long as you can reach it on the internet.

Purple906 commented:
@mysterycommand not sure what the fail error is now. I thought the reason this failed the first time is because the branch was referencing localhost:8888. 

I just updated to a live version of WP, imported/recreate the content, and updated the codebase on branch "update-stories-page" to reflect the correct URL. Everything seemed to be working fine when I ran Yarn-dev. So I was then able to commit and merge the changes from update-stories-page into updates-3 branch (Said no conflicts).

So I thought that would bring this branch up to snuff and I would be able to merge it into master. What am I missing?

mysterycommand commented:
Okay, I did it … I think it should build now. Also while I was in there I removed the old `matts-page.js` and cleaned up a `console.log` and removed some unused imports. Just tidying up. You can step through my commits above one at a time (by clicking the little code on the right) if you want to look at each "step" … also you can run a build locally with `yarn export` if you ever want to try and debug a build locally in the future.
